apps/testnorge-arena/arena-core/src/main/java/no/nav/registre/arena/core/service/VedtakshistorikkService.java

```diff
@@ -180,14 +180,17 @@ private LocalDate settTidligeDatoBarnetillegg(List<NyttVedtakTiltak> barnetilleg
 
         List<RettighetRequest> rettighetRequests;
 
-        if (senesteVedtak instanceof NyttVedtakAap) {
+        if (senesteVedtak == null){
+            log.info("Kunne ikke opprette rettigheter for ident: " + personident);
+            rettighetRequests = new ArrayList<>();
+        }else if (senesteVedtak instanceof NyttVedtakAap) {
```

Comment:
mangler mellomrom :P
Ellers vil jeg legge på en fair warning om at å bruke `instanceof` er sett på som litt [dårlig stil](https://stackoverflow.com/questions/2750714/is-instanceof-considered-bad-practice-if-so-under-what-circumstances-is-instan). I dette tilfelle er det absolutt ikke alt for ille, men man kan heller ha et felles interface, der man evt. har en metode for type som blir implementert:

```java
public interface NyttVedtak {
    public Vedtakstype get_vedtakstype();
}

public enum Vedtakstype {
    AAP,
    TILTAK,
    TILLEGG
}
```
Bruker du interface mener jeg du også kan switche på sensteVedtak ;)
Evt. fant jeg et [svar](https://stackoverflow.com/a/29570906/9502132) som løser dette på en helt annen måte.

Reply:
God ide, skal se om jeg kan få oppdatert det :)